extensions/sql-migration/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "sql-migration",
 	"displayName": "%displayName%",
 	"description": "%description%",
-	"version": "0.1.13",
```

Comment:
i have already bumped from 0.1.12  (production) to 0.1.13 after last release.

Reply:
I'll close this pr for now. thanks